go/libraries/doltcore/rebase/rebase_tag.go

```diff
@@ -234,6 +351,12 @@ func replayCommitWithNewTag(ctx context.Context, root, parentRoot, rebasedParent
 			return nil, err
 		}
 
+		rebasedParentRows, err = dropValsForDeletedColumns(ctx, root.VRW().Format(), rebasedParentRows, rebasedSch, rebasedParentSch)
```

Comment:
Do we do this in the general case of dropping a column? We didn't use to

Reply:
We do, yes. https://github.com/liquidata-inc/dolt/pull/386